tests/functional/tests/bundles/overlays/local-charm-overlay.yaml.j2

```diff
@@ -0,0 +1,10 @@
+applications:
+  {{ charm_name }}:
+    charm: "{{ CHARM_LOCATION }}/{{ charm_name }}.charm"
+    num_units: 0
+    resources:
+      exporter-snap: "resources/software-inventory-exporter.snap"
```

Comment:
It's fine to use local resource for development of tests before the snap is in the store, but it can't be pushed to the main repo and used in CI.